client/src/pages/ModalTestingPage.js

```diff
@@ -2,26 +2,89 @@ import React, { useState } from "react";
 import Button from "react-bootstrap/Button";
 import ModalVideo from "../components/ModalVideo";
 
-function ModalTestingPage(){
-    const [show, setShow] = useState(false);
-    const handleClose = () => setShow(false);
-    const handleShow = () => setShow(true);
-    const data =
-        {
-            title: "Rumba Guaguanco",
-            url: "https://www.youtube.com/watch?v=gJVT_5swkhA",
-            location: "Havana, Cuba",
-        };
+const rhythms = [
+	{
+		title: "DC Hand dancing",
+		url: "https://www.youtube.com/watch?v=M6uM0qrjetQ",
+		location: "Washington DC",
+		description: "I have both Video and Description",
+	},
+	{
+		title: "Rumba Guaguanco",
+		location: "Havana, Cuba",
+		description: "I have no video to play, for more information google me!",
+	},
+	{
+		title: "DC Hand dancing 2",
+		url: "https://www.youtube.com/watch?v=M6uM0qrjetQ",
+		location: "Washington DC",
+	},
+	{
+		title: "Adaha and Konkoma",
+		audioUrl:
+			"https://soundcloud.com/shugo_srd/shugz-king-of-my-castle?utm_source=clipboard&utm_medium=text&utm_campaign=social_sharing",
+		location: "Washington DC",
+	},
+	{
+		title: "Dance Band Highlife",
+		audioUrl:
+			"https://soundcloud.com/eduleon42/are-we-going-out-tonight-vol-3?si=0cd23cdd892b41c6ad451505b21c0d3f&utm_source=clipboard&utm_medium=text&utm_campaign=social_sharing",
+		location: "Washington DC",
+		description:
+			"Dance band likely had its immediate roots in the orchestras founded and popularized by West African coastal elites who lived in the region’s expanding colonial and cosmopolitan cities",
+	},
+];
```

Comment:
I mentioned this on Slack, but please let me know what work is needed to integrate this data into our API.